conformance/reports/v1.0.0/projectcontour-contour.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
```

Comment:
Recommend updating implementations page to link to this report as well (https://gateway-api.sigs.k8s.io/implementations/#contour)

Reply:
good catch, done!